javaslang/src/test/java/javaslang/collection/LinkedHashMultimapTest.java

```diff
@@ -21,7 +20,7 @@ protected String className() {
     }
 
     @Override
-    <T1, T2> Map<T1, T2> javaEmptyMap() {
+    <T1, T2> java.util.LinkedHashMap<T1, T2> javaEmptyMap() {
```

Comment:
\[Checkstyle\] ERROR: Declaring variables, return values or parameters of type 'java\.util\.LinkedHashMap' is not allowed\.